<launch>

 <!-- joy node -->
  <node respawn="false" pkg="joy" type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js0" /> 
    <!-- <param name="deadzone" value="0.12" /> -->
    <param name="autorepeat_rate" value="10.0" />
  </node>

 <!-- Axes -->

  <node pkg="vrep_ros_teleop" type="teleop.py" name="teleop">
      <param name="axis_linear" value="1" type="int"/>
      <param name="axis_angular" value="0" type="int"/>
      <param name="scale_linear" value="1.0" type="double"/>
      <param name="scale_angular" value="-1.0" type="double"/>
      <remap from="/teleop/joy" to="/joy"/>
  </node>

  <node pkg="topic_tools" type="mux" name="cmd_mux" 
      args="/cmd_vel_mux/input/navi /teleop/twistCommand /mux/autoCommand">
  </node>

  <node pkg="vrep_ros_teleop" type="teleop_mux.py" name="teleop_mux">
      <param name="joystick_button" value="0" type="int"/>
      <param name="joystick_topic" value="/teleop/twistCommand"/>
      <param name="auto_button" value="1" type="int"/>
      <param name="auto_topic" value="/mux/autoCommand"/>
      <remap from="/teleop_mux/joy" to="/joy"/>
  </node>

</launch>
